spring-cloud-skipper-server/src/test/java/org/springframework/cloud/skipper/server/controller/docs/ApiDocumentation.java

```diff
@@ -68,12 +67,28 @@ public void errors() throws Exception {
 	@Test
 	public void index() throws Exception {
 		this.mockMvc.perform(get("/")).andExpect(status().isOk()).andDo(this.documentationHandler.document(links(
+			linkWithRel("about").description("Provides meta information of the server"),
+			linkWithRel("upload").description("Upload a package"),
+			linkWithRel("install").description("Install a package"),
+			linkWithRel("install-with-package-id").description("Install a package by also providing the package id"),
 			linkWithRel("appDeployerDatas").description("Exposes App Deployer Data"),
 			linkWithRel("repositories").description("Exposes package repositories"),
 			linkWithRel("deployers").description("Exposes deployer"),
 			linkWithRel("releases").description("Exposes release information"),
 			linkWithRel("packageMetadata").description("Provides details for Package Metadata"),
-			linkWithRel("profile").description("Entrypoint to provide ALPS metadata defining simple descriptions of application-level semantics.")
+			linkWithRel("profile").description("Entrypoint to provide ALPS metadata defining simple descriptions of application-level semantics"),
+
+			linkWithRel("status").description("Status for a last known release version"),
+			linkWithRel("status-by-name-and-version").description("Status for a last known release version by release name"),
```

Comment:
This needs to be for the release of the given name and version. Will fix while merging.